core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/TurtleParser.java

```diff
@@ -70,6 +70,7 @@
 	protected Value object;
 
 	private int lineNumber = 1;
+	private StringBuilder parsingBuilder = new StringBuilder();
```

Comment:
This should be declared as final to ensure that it is recognised as a permanent placeholder that can't be accidentally reassigned.